pkg/acsengine/defaults-kubelet.go

```diff
@@ -97,17 +98,21 @@ func setKubeletConfig(cs *api.ContainerService) {
 		}
 		setMissingKubeletValues(cs.Properties.MasterProfile.KubernetesConfig, o.KubernetesConfig.KubeletConfig)
 		addDefaultFeatureGates(cs.Properties.MasterProfile.KubernetesConfig.KubeletConfig, o.OrchestratorVersion, "", "")
-
 	}
+
 	// Agent-specific kubelet config changes go here
 	for _, profile := range cs.Properties.AgentPoolProfiles {
 		if profile.KubernetesConfig == nil {
 			profile.KubernetesConfig = &api.KubernetesConfig{}
 			profile.KubernetesConfig.KubeletConfig = copyMap(profile.KubernetesConfig.KubeletConfig)
 		}
 		setMissingKubeletValues(profile.KubernetesConfig, o.KubernetesConfig.KubeletConfig)
-		if !common.IsKubernetesVersionGe(o.OrchestratorVersion, "1.11.0") {
-			addDefaultFeatureGates(profile.KubernetesConfig.KubeletConfig, o.OrchestratorVersion, "1.6.0", "Accelerators=true")
+
+		// For N Series (GPU) VMs
+		if strings.Contains(profile.VMSize, "Standard_N") {
+			if !cs.Properties.IsNVIDIADevicePluginEnabled() { // enabling accelerators for Kubernetes >= 1.6 to <= 1.9
```

Comment:
Is this comment true that this is not expected to engage for 1.10.0 and 1.11.0?

Reply:
that's right, we don't want `accelerators` flag for NVIDIA device plugin. 
Added a check for v1.11 also, just in case user has disabled `nvidia-device-plugin` explicitly so cluster should still be up but there will not be any GPUs available for K8s.